examples/48-drawindirect/cs_drawindirect.sc

```diff
@@ -14,19 +14,25 @@ BUFFER_WR(instanceBufferOut, vec4, 2);
 
 uniform vec4 u_drawParams;
 
-NUM_THREADS(1, 1, 1)
+NUM_THREADS(1024, 1, 1)
```

Comment:
What results you get with 64 wide?

Reply:
you should change dispatch...

`bgfx::dispatch(maxToDraw/64, m_indirect_program);`

and then do

`NUM_THREADS(64, 1, 1)`

Reply:
Same result

Reply:
Changes have been applied